src/mame/atari/atarittl.cpp

```diff
@@ -297,11 +312,21 @@ void gtrak10_state::gtrak10(machine_config &config)
 	m_video->set_vert_params( GTRAK10_VTOTAL - 8,   GTRAK10_VTOTAL - 8,       GTRAK10_VTOTAL,     GTRAK10_VTOTAL);
 	m_video->set_fieldcount(2);
```

Comment:
Are you sure that the video output is interlaced? The logic in this case must generate different timings for the two fields.

Reply:
The interlace detection (fieldcount=2) I wrote from specifications if I remember it right. I really need to see the `compsync` signal and check the code against it. 
This PR is a significant step forward. I am happy to commit it once all checks have passed.

Reply:
Cool!  It's kinda fun when you can just hook things up virtually with a text description and there's a complete set of logic chips to plug in and connect like a breadboard.  Like the 555 timer, I just copied an example from JD's tank driver as a model.

Reply:
yep, @goldnchild , that's exactly how it's intended to work, just like a breadboard, or rather, like Lego! The old "external netlist as ROM" (the old "pong" as opposed to "pongf") in theory made it even more so with less compiling - in practice, ehhhh. The pain also come during development when the component you want isn't in the library of components, or if your netlist is so non-performant that you have to play around with static solvers and @couriersud 's other performance optimizing features...

Reply:
I was thinking of a hybrid approach where the sound can be done with netlist and the video generation can be done with "regular mame code".  My computer isn't fast enough for netlist to work in the multi megahertz range at full speed, but I reckon sound is at least an order of magnitude less demanding.  (khz vs megahertz)